recipes/python3/ModulesSetup

```diff
@@ -2,9 +2,11 @@
 # Static compilation instructions for all binary modules.
 #####################################################################
 
+*static*
 _asyncio _asynciomodule.c
 _bisect _bisectmodule.c
 _blake2 _blake2/blake2module.c _blake2/blake2b_impl.c _blake2/blake2s_impl.c
+_sqlite3 -I$(srcdir)/Modules/_sqlite -DMODULE_NAME='\"sqlite3\"' _sqlite/cache.c _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c
```

Comment:
Why the custom line doesn't include -DSQLITE_OMIT_LOAD_EXTENSION?

Reply:
Is something by default, If We're more confident about having it in our `ModulesSetup` I can revert this file completely.

Reply:
I can confirm that reverting `ModulesSetup` to previous version leads to the same result.